torchgeo/models/swin.py

```diff
@@ -46,75 +46,138 @@
 Weights.__deepcopy__ = lambda *args, **kwargs: args[0]
 
 
-class Swin_V2_B_Weights(WeightsEnum):  # type: ignore[misc]
-    """Swin Transformer v2 Base weights.
+class Swin_V2_T_Weights(WeightsEnum):  # type: ignore[misc]
+    """Swin Transformer v2 Tiny weights.
 
     For `torchvision <https://github.com/pytorch/vision>`_
-    *swin_v2_b* implementation.
+    *swin_v2_t* implementation.
 
     .. versionadded:: 0.6
     """
 
-    NAIP_RGB_SI_SATLAS = Weights(
-        url='https://hf.co/allenai/satlas-pretrain/resolve/daa578a4be36573d9791bf51dcd0420b8dc75732/aerial_swinb_si.pth',
+    SENTINEL2_MI_MS_SATLAS = Weights(
+        url='https://hf.co/torchgeo/satlas/resolve/081d6607431bf36bdb59c223777cbb267131b8f2/sentinel2_swint_mi_ms-d8c659e3.pth',
+        transforms=_satlas_sentinel2_transforms,
+        meta={
+            'dataset': 'SatlasPretrain',
+            'in_chans': 9,
+            'model': 'swin_v2_t',
+            'publication': 'https://arxiv.org/abs/2211.15660',
+            'repo': 'https://github.com/allenai/satlas',
+            'bands': ('B02', 'B03', 'B04', 'B05', 'B06', 'B07', 'B08', 'B11', 'B12'),
+        },
+    )
+
+    SENTINEL2_MI_RGB_SATLAS = Weights(
+        url='https://hf.co/torchgeo/satlas/resolve/081d6607431bf36bdb59c223777cbb267131b8f2/sentinel2_swint_mi_rgb-424d91f4.pth',
         transforms=_satlas_transforms,
         meta={
-            'dataset': 'Satlas',
+            'dataset': 'SatlasPretrain',
             'in_chans': 3,
-            'model': 'swin_v2_b',
+            'model': 'swin_v2_t',
+            'publication': 'https://arxiv.org/abs/2211.15660',
+            'repo': 'https://github.com/allenai/satlas',
+            'bands': ('B02', 'B03', 'B04'),
```

Comment:
Can anyone confirm if the band order is RGB or BGR?

Reply:
According to their [demo](https://github.com/allenai/satlaspretrain_models/blob/main/demo.ipynb), it looks more like RGB. I haven't found a clear definition of TCI order, but it seems to be RGB.
@favyen2 @piperwolters, please confirm.

Reply:
It should be RGB..
ESA’s Sentinel-2 User Handbook: The TCI product is described as a true color composite using Red (Band 4), Green (Band 3), and Blue (Band 2) bands.

Reply:
[Satlas Normalization README](https://github.com/allenai/satlas/blob/55bb52fb90a9caaa91bf5fb428e76bfd911a5364/Normalization.md#sentinel-2-images)

Reply:
So this is also true for MSI? It would be 4, 3, 2, 5, 6, 7, ...? @yichiac I'm curious if this is the order you used in your experiments or not. Could explain why SatlasPretrain performed worse than SSL4EO-S12 (or it could be the subset of supported bands).

Reply:
NAIP too?

Reply:
I think TCI's order is 4, 3, 2, and MSI is 2, 3, 4, 5, 6, 7, ... The starting accuracy (acc for the first few epochs) of SatlasPretrain is usually the highest.

Reply:
NAIP is also RGB. 

I'm pretty sure the MSI is TCI, 5, 6, 7, 8, 11, 12, so 4, 3, 2, 5, 6, 7, 8, 11, 12. 
See [this Sentinel-2 example](https://github.com/allenai/satlas/blob/main/Normalization.md#sentinel-2-example) in Satlas docs.

Reply:
@yichiac we should redo our Satlas experiments with the correct band order.